llvm/lib/Target/X86/MCTargetDesc/X86MCCodeEmitter.cpp

```diff
@@ -948,10 +948,10 @@ X86MCCodeEmitter::emitVEXOpcodePrefix(int MemOperand, const MCInst &MI,
     llvm_unreachable("Unexpected form in emitVEXOpcodePrefix!");
   case X86II::MRMDestMem4VOp3CC: {
     //  MemAddr, src1(ModR/M), src2(VEX_4V)
+    Prefix.setR(MI, CurOp++);
```

Comment:
Don't understand the change. Is the `CurOp` the index of operands?
According to above comment and assemble test, the order in `MemAddr, src1(ModR/M), src2(VEX_4V)` was correct. Does the change make it `src1(ModR/M), MemAddr, src2(VEX_4V)`?

Reply:
Yes, I think so. The comment at line 950 needs to be corrected too.

Reply:
You are correct. addressed in [1e66d6c](https://github.com/llvm/llvm-project/pull/66748/commits/1e66d6cc36130cd9d66e2233b3a78c3ca4222721)